src/Shared/Microsoft.DotNet.Kusto/KustoHelpers.cs

```diff
@@ -154,8 +154,13 @@ public static ClientRequestProperties BuildClientRequestProperties(KustoQuery qu
             properties.SetParameter(parameter.Name, parameter.Value.ToString());
         }
 
-        return properties;
-    }
+            foreach (var option in query.Options)
+            {
+                properties.SetOption(option.Name, option.Value);
```

Comment:
I really wish we wrapped this with known values, rather than exposing this arbitrary dictionary. It breaks the abstraction pretty hard, since now you need to go dig into Kusto code to figure out what these "options" are and what value they are adding.

Do we _really_ need to set all of these?  Can we have a "KustoQueryOptions" class that wraps the few we know about/care about/support, so that our Helper API is a bit clearer to use?